src/components/notifications/notifications.js

```diff
@@ -248,7 +248,7 @@ events.on(serverNotifications, 'RestartRequired', function (e, apiClient) {
         [
             {
                 action: 'restart',
-                title: globalize.translate('ButtonRestart'),
+                title: globalize.translate('HeaderRestart'),
```

Comment:
Above the one that was picked was the Button* prefix, and here it's the Header* prefix. Maybe the Button prefix is more general.

Reply:
fixed